config/jobs/kubernetes/sig-gcp/sig-gcp-gce-config.yaml

```diff
@@ -177,6 +177,44 @@ presubmits:
           requests:
             memory: "6Gi"
 
+  - name: pull-kubernetes-e2e-gce-csi-serial
```

Comment:
wonder if it should live under sig-storage?

Reply:
sure will add

Reply:
I mean this config is now under sig-gcp and you won't able to further approve in the future :-)